src/test/scala/io/iohk/ethereum/validators/BlockValidatorSpec.scala

```diff
@@ -46,12 +46,6 @@ class BlockValidatorSpec extends FlatSpec with Matchers {
     }
   }
 
-  "Block" should "return a block when verifing if a block body corresponds to a block header" in {
```

Comment:
Minor detail: Why was this test removed?

Reply:
We are no longer returning `block` from `validateHeaderAndBody` and test was explicitly testing this. A valid case of a validator is covered by other tests.